src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -15,6 +15,13 @@
 				disable-tooltip />
 		</template>
 		<h4 class="absence-reminder__caption">{{ userAbsenceCaption }}</h4>
+		<div v-if="hasReplacementUser" class="absence-reminder__replacement">
+			<p>{{ t('spreed','Replacement: ') }}</p>
+			<NcUserBubble class="absence-reminder__replacement__bubble"
+				:display-name="userAbsence.replacementUserDn"
+				:user-id="userAbsence.replacementUserId"
+				@click="openConversationWithReplacementUser(userAbsence.replacementUserId)" />
```

Comment:
Since you're not using an argument in the method:
```suggestion
				@click="openConversationWithReplacementUser" />
```